README.rst

```diff
@@ -30,11 +30,15 @@ OpenStack Configuration
 
 You must update the ``plugin_dirs`` parameter in the ``heat.conf`` file
 to include the resources from this module. Typically this would mean
-adding ``/usr/local/lib/python2.7/dist-packages/heat_infoblox``:
-
+adding ``/usr/local/lib/python2.7/dist-packages/heat_infoblox``.
+Also you must add ``lock_path`` under ``oslo_concurency`` stanza.
+For security, the specified directory should only be writable by the user
+running the heat process:
 ::
 
   plugin_dirs = /usr/local/lib/python2.7/dist-packages/heat_infoblox,/usr/lib64/heat,/usr/lib/heat
+  [oslo_concurrency]
+  lock_path = /tmp/
```

Comment:
Well, if it only should be writable by that user, then you shouldn't use /tmp as an example!